tests/Services/SearchServiceTest.php

```diff
@@ -229,7 +242,7 @@ public function testRedirectToProductPageOnDoSearch(
         array $attributes,
         string $language = 'de',
         string $defaultLanguage = 'de',
-        bool $isOptionShowPleaseSelectEnabled = true
+        bool $optionShowPleaseSelect = true
```

Comment:
I think I mentioned this in the last review with Modestas. The thing is the naming itself "optionShowPleaseSelect" should be renamed, as a boolean should typically make sure to expose his state by making it clear what happens if that state is reached. Something like `isEnabled` or `hasSticker` are pretty obvious namings, as they refer directly to the state and it is easy to understand what happens if their state changes from `true` to `false` or vice versa. 

I am fine with this for now, but we should definitely keep an eye out for that in the future.